src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -217,37 +208,30 @@ private void validateNewOrUpdatedFileUploadExercise(FileUploadExercise fileUploa
     }
 
     /**
-     * GET /file-upload-exercises/:exerciseId : get the "id" fileUploadExercise.
+     * GET file-upload-exercises/:exerciseId : get the fileUploadExercise with id exerciseId.
```

Comment:
```suggestion
     * GET file-upload-exercises/:exerciseId : get the FileUploadExercise with id exerciseId.
```
To be consistent with the refactoring further up in the class.